lib/available_delivery_location_types.js

```diff
@@ -17,7 +17,7 @@ class AvailableDeliveryLocationTypes {
     let apiURL = `patrons/${patronID}`
     return client.get(apiURL).then((response) => {
       logger.debug(`response from patron service ${apiURL}: ${JSON.stringify()}`)
```

Comment:
I mentioned this in the post-mortem PR but this `JSON.stringify()` caused us some confusion when were trying to debug this issue because it is a stringifying nothing. Maybe we should remove or add some other helpful, non-PII information?